pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/MembershipManager.java

```diff
@@ -225,7 +232,8 @@ public void checkFailures(FunctionMetaDataManager functionMetaDataManager,
             functionRuntimeManager.removeAssignments(needRemove);
         }
         if (triggerScheduler) {
-            log.info("Functions that need scheduling/rescheduling: {}", needSchedule);
+            log.info("Failure check - Total number of instances that need to be scheduled/rescheduled: {} | Number of unassigned instances that need to be scheduled: {} | Number of instances on dead workers that need to be reassigned {}",
```

Comment:
Maybe a better wording here?

Reply:
Do you have a suggestion?

Reply:
I actually liked the old message. What was the reason for the change?

Reply:
Added "Failure check" in the beginning so it will be easier to search for